mistletoe/latex_renderer.py

```diff
@@ -51,9 +56,9 @@ def render_escape_sequence(self, token):
         return self.render_inner(token)
 
     def render_raw_text(self, token, escape=True):
-        return (token.content.replace('$', '\$').replace('#', '\#')
-                             .replace('{', '\{').replace('}', '\}')
-                             .replace('&', '\&')) if escape else token.content
+        return (token.content.replace('$', '\\$').replace('#', '\\#')
+                             .replace('{', '\\{').replace('}', '\\}')
+                             .replace('&', '\\&')).replace('_', '\\_') if escape else token.content
```

Comment:
Yes, this seems OK, but what `%`, it looks like it should be escaped as well. I'm no expert on LaTeX though, so don't hesitate to correct me.